diff --git a/config.h.cmake b/config.h.cmake
index cc4fbfd..58da8ed 100644
--- a/config.h.cmake
+++ b/config.h.cmake
@@ -2,3 +2,4 @@
 #cmakedefine WITH_TRANSLATIONS
 #cmakedefine LINUX_DIST_FLATPAK
 #cmakedefine BUNDLE_QT_TRANSLATIONS
+#cmakedefine CMAKE_INSTALL_LIBDIR "@CMAKE_INSTALL_LIBDIR@"
diff --git a/src/app/main.cpp b/src/app/main.cpp
index 1aa2864..141a0a2 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -62,7 +62,7 @@ int main(int argc, char *argv[]) {
 #ifdef Q_OS_LINUX
   QDir appPath = a.applicationDirPath();
   appPath.cdUp();
-  appPath.cd("lib");
+  appPath.cd(CMAKE_INSTALL_LIBDIR);
   a.addLibraryPath(appPath.filePath("sane-break"));
   LinuxSystemSupport::check();
 #endif  // Q_OS_LINUX
